core/controlplane/config/templates/cluster.yaml

```diff
@@ -1094,6 +1094,14 @@ worker:
 #kubernetes:
 #  networking:
 #    selfHosting:
+#  controllerManager:
+#    resources:
+#      requests:
+#        cpu: 200m
```

Comment:
Usually the values here are the defaults. Are there any recommended defaults from upstream?

Reply:
This is the default value righ know for cpu. That I added for limits are based on our production cluster requirements. If you know where I can find any recommendation for this, please, share

Reply:
Let's continue the discussion in #1254